notebooks/azureml_model_headers_demo/german_credit_azure_ml_demo.ipynb

```diff
@@ -1398,195 +829,9 @@
-      "text/plain": [
-       "<Figure size 864x288 with 1 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    }
-   ],
+   "execution_count": null,
```

Comment:
This notebook has been saved without output.  Previously the output was included so that people could scan through and see results without having to run it themselves.  Is this a deliberate decision to change that?

Reply:
You are correct I fixed it.